swarm-test/src/lib.rs

```diff
@@ -235,7 +235,7 @@ where
         let peer_id = PeerId::from(identity.public());
 
         let transport = MemoryTransport::default()
-            .or_transport(libp2p_tcp::async_io::Transport::default())
+            .or_transport(libp2p_tcp::tokio::Transport::default())
```

Comment:
this function, `new_ephemeral` is still feature gated to `async-std`  and we probably should remove `new_ephemeral_tokio` as `new_ephemeral` by default uses tokio

Reply:
yeah thanks Elena, but then this should still use `async_io` then right?